apps/main/src/dex/components/PagePool/Swap/index.tsx

```diff
@@ -381,6 +381,7 @@ const Swap = ({
                 disabled={isDisabled || selectList.length === 0}
                 showSearch={false}
                 showManageList={false}
+                compact={true}
```

Comment:
compact is optional bool, the true could be ommitted
```suggestion
                compact
```